bsb/morphologies/__init__.py

```diff
@@ -1248,6 +1269,60 @@ def attach_child(self, branch):
         self._children.append(branch)
         branch._parent = self
 
+    def find_closest_point(self, coord):
+        """
+        Return the index of the closest on this branch to a desired coordinate.
+
+        :param coord: The coordinate to find the nearest point to
+        :type: :class:`numpy.ndarray`
+        """
+        diff = np.sqrt(np.sum((self._points - coord) ** 2, axis=1))
+        return np.argmin(diff)
+
+    def insert_branch(self, branch, index):
+        """
+        Insert a new branch, by splitting this branch and inserting ``branch`` at a specific index.
```

Comment:
```suggestion
        Split this branch and insert the given ``branch`` at the specified ``index``.
```

I think the explanation of `:param index:` is good!